docs: twitter elevated access information on twitter auth page (#11224)
supabase/supabase: apps/docs/pages/guides/auth/social-login/auth-twitter.mdx

  description: 'Add Twitter OAuth to your Supabase project',
}

> To enable Twitter Auth for your project, you need to set up a Twitter OAuth application and add the application credentials to your Supabase Dashboard. You will also need an [elevated access](https://developer.twitter.com/en/portal/products/elevated) from Twitter.

Reviewer: ```suggestion
To enable Twitter Auth for your project, you need to set up a Twitter OAuth application with [elevated access](https://developer.twitter.com/en/portal/products/elevated)and add the application credentials in the Supabase Dashboard.
```
